### DISCUSSION TITLE
Resume failed rips on interrupt (docker restart or accidental disk eject)?

### DISCUSSION THREAD

**Discussion by ghost** [automatic-ripping-machine/automatic-ripping-machine#1237] [Ideas]
Is it possible to make ARM more gracefully handle things like the docker container being restarted (due to server crash or using docker compose up on a long configuration file) or a disk being ejected?

Like if ARM detects it was restarted it should refresh the web gui with the correct status, and possibly try to resume from the point of failure. If a disk was accidentally ejected and later inserted, it should try to resume the failed rip (once verifying that the disk is 100% correct)

**Comment by srice555** [comment]
I find it shocking that the team built a really nice software suite, but absolute zero ability to restart where it left off.
